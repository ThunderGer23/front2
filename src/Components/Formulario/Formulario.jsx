export const Formulario = () =>{
    return (
        <>
        <div role='div'>
            <form role='form' action="" className="bt-4">
                
            </form>
        </div>
        </>
    )
}