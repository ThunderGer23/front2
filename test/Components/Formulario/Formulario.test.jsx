import { render, screen } from "@testing-library/react";
import { Formulario } from "../../../src/Components/Formulario/Formulario";

describe('Pruebas en el componente Formulario', () => { 
    test('Esta prueba es para hacer match con el snapshot', () => { 
        const {container} = render(<Formulario/>)//Renderiza el componente Formulario
        expect(container).toMatchSnapshot()//Toma una foto
     })

    test('Esta prueba valida que el div exista', () => { 
        render(<Formulario/>)
        expect(screen.findAllByRole('div')).toBeTruthy()//Se calida que exista el div
     })

    test('Esta prueba valida que el form exista y pertenezca a la clase correcta', () => { 
        render(<Formulario/>)
        expect(screen.findByRole('form')).toBeTruthy()
        const {className} = screen.getByRole('form')//Se salva la clase del div del componente en className
        expect(className).toBe('bt-4')
     })
})