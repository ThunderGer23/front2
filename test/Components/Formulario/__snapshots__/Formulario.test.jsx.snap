// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente Formulario Esta prueba es para hacer match con el snapshot 1`] = `
<div>
  <div
    role="div"
  >
    <form
      action=""
      class="bt-4"
      role="form"
    />
  </div>
</div>
`;
